server.py

```diff
@@ -39,21 +41,15 @@ def outgoing():
   resp.say("Congratulations! You have made your first oubound call! Good bye.")
   return str(resp)
 
-@app.route('/incoming', methods=['GET', 'POST'])
-def incoming():
-  resp = twilio.twiml.Response()
-  resp.say("Congratulations! You have received your first inbound call! Good bye.")
-  return str(resp)
-
 @app.route('/placeCall', methods=['GET', 'POST'])
 def placeCall():
-  account_sid = os.environ.get("ACCOUNT_SID", ACCOUNT_SID)
-  api_key = os.environ.get("API_KEY", API_KEY)
-  api_key_secret = os.environ.get("API_KEY_SECRET", API_KEY_SECRET)
-
-  client = Client(api_key, api_key_secret, account_sid)
-  call = client.calls.create(url=request.url_root + 'incoming', to='client:' + IDENTITY, from_='client:' + CALLER_ID)
-  return str(call.sid)
+    response = twilio.twiml.Response()
+    to = request.form.get('server_param_to', IDENTITY)
+    if to[0] in "+1234567890":
```

Comment:
Add a comment here. Something like `For simplicity, the following code assumes that identity name starts only with letters and not with numbers.`